analytics/analytics.go

```diff
@@ -19,6 +19,8 @@ import (
 	"gorm.io/gorm/clause"
 )
 
+const flushInterval = time.Hour
```

Comment:
Non blocker: We should eventually parameterise this.

Reply:
👍 There are a number of similar values throughout singularity.

Reply:
https://github.com/data-preservation-programs/singularity/issues/421